File: C++/vetores/exercicios/idades.cpp
#include <iostream>
#include <string>
#include <locale.h>
#include <iomanip>
using namespace std;

int main(){
	int qtd, i;
	float media;
	setlocale(LC_ALL, "");
	
	cout << "Digite a quantidade de pessoas: ";
	cin >> qtd;
	
	int idades[qtd], somaIdades=0;
	string nomes[qtd];
	
	for(i=0;i<qtd;i++){
	    cout << "\nDigite o " << i+1 << "º nome: ";
	    cin >> ws; 
        getline(cin, nomes[i]);
	    
		cout << "\nDigite a idade de " << nomes[i] << ":";
		cin >> idades[i];
		
		somaIdades += idades[i];
	}
	media = somaIdades / qtd;
	
	cout << "Média das idades: " << media << endl;
	for(i=0;i<qtd;i++){
	    if(idades[i] > media){
	    	cout << nomes[i] << " possui a idade acima da média. Idade: " << idades[i];
		}
	}
	return 0;
}

File: Python/idades.py
qtd = int(input("Digite a quantidade de pessoas: "))

i=0
idades = []
somaIdades=0
nomes = []

for i in range(qtd):
    nomes.append(input(f"Digite o {i+1} nome: "))
    idades.append(int(input(f"\nDigite a idade de {nomes[i]}: ")))

somaIdades = sum(idades)
print(somaIdades)

media = somaIdades / qtd
i=0
print(f"Média de idades: {media}")

def maiorqueMedia():
    if idades[i] > media:
        print(f"\n{nomes[i]} possui a idade maior que a média.")


for i in range(qtd):
    maiorqueMedia()
